### PR TITLE
[US-3593] Shopsys framework now uses latest version of Shopsys coding standards

### PR DESCRIPTION
**Pull request by boris-brtan**
| Q             | A
| ------------- | ---
|Description, reason for the PR| ...
|New feature| Yes
|BC breaks| Yes
|Fixes issues| -
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/coding-standards/shopsys-coding-standard.yml [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+        PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\EmptyStatementSniff.DetectedCatch: ~
+
+        ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
+        # skip known issues for framework
+            - packages/framework/src/DataFixtures/Base/MailTemplateDataFixture.php
+            - packages/framework/src/DataFixtures/Demo/CategoryDataFixture.php
+            - packages/framework/src/DataFixtures/Demo/OrderDataFixture.php
+            - packages/framework/src/DataFixtures/DemoMultidomain/CategoryDataFixture.php
+            - packages/framework/src/DataFixtures/DemoMultidomain/MailTemplateDataFixture.php
+            - packages/framework/src/DataFixtures/DemoMultidomain/OrderDataFixture.php
```
Comment (TomasVotruba) [inline packages/coding-standards/shopsys-coding-standard.yml]: This can be actually simplified by using fnmatch() format, see docs: https://github.com/symplify/easyCodingStandard/#ignore-what-you-cant-fix

```diff
 parameters: 
     skip:
         ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
             # skip known issues for framework
-            - packages/framework/src/DataFixtures/Base/MailTemplateDataFixture.php
-            - packages/framework/src/DataFixtures/Demo/CategoryDataFixture.php
-            - packages/framework/src/DataFixtures/Demo/OrderDataFixture.php
-            - packages/framework/src/DataFixtures/DemoMultidomain/CategoryDataFixture.php
-            - packages/framework/src/DataFixtures/DemoMultidomain/MailTemplateDataFixture.php
+            - "*packages/framework/src/DataFixtures/*/*DataFixture.php"
```

File: packages/coding-standards/shopsys-coding-standard.yml [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+            - packages/framework/src/Form/Admin/Category/CategoryFormType.php
+            - packages/framework/src/Form/Admin/Customer/BillingAddressFormType.php
+            - packages/framework/src/Form/Admin/Customer/DeliveryAddressFormType.php
+            - packages/framework/src/Form/Admin/Customer/UserFormType.php
+            - packages/framework/src/Form/Admin/Order/OrderFormType.php
+            - packages/framework/src/Form/Admin/Payment/PaymentFormType.php
+            - packages/framework/src/Form/Admin/Product/ProductEditFormType.php
+            - packages/framework/src/Form/Admin/Product/ProductFormType.php
+            - packages/framework/src/Form/Admin/Slider/SliderItemFormType.php
+            - packages/framework/src/Form/Admin/Transport/TransportFormType.php
```
Comment (TomasVotruba) [inline packages/coding-standards/shopsys-coding-standard.yml]: Same trick can be used here, I guess you get the idea. 

I personally use them when 3+ files with simliar name format reports error from the same checker service.

```diff
 parameters:
     skip:
         ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
-            - packages/framework/src/Form/Admin/Administrator/AdministratorFormType.php
-            - packages/framework/src/Form/Admin/Advert/AdvertFormType.php
-            - packages/framework/src/Form/Admin/Article/ArticleFormType.php
-            - packages/framework/src/Form/Admin/Category/CategoryFormType.php
-            - packages/framework/src/Form/Admin/Customer/BillingAddressFormType.php
-            - packages/framework/src/Form/Admin/Customer/DeliveryAddressFormType.php
-            - packages/framework/src/Form/Admin/Customer/UserFormType.php
-            - packages/framework/src/Form/Admin/Order/OrderFormType.php
-            - packages/framework/src/Form/Admin/Payment/PaymentFormType.php
-            - packages/framework/src/Form/Admin/Product/ProductEditFormType.php
-            - packages/framework/src/Form/Admin/Product/ProductFormType.php
-            - packages/framework/src/Form/Admin/Slider/SliderItemFormType.php
-            - packages/framework/src/Form/Admin/Transport/TransportFormType.php
+            - "*/packages/framework/src/Form/Admin/*/*FormType.php"
```

File: packages/framework/src/DataFixtures/Base/SettingValueDataFixture.php [hunk 3]
```diff
@@ -49,7 +49,6 @@ public function __construct(
 
     /**
      * @param \Doctrine\Common\Persistence\ObjectManager $manager
-     * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
```
Comment (TomasVotruba) [inline packages/framework/src/DataFixtures/Base/SettingValueDataFixture.php]: Beautiful :heart:

File: packages/framework/src/DataFixtures/DemoMultidomain/OrderDataFixture.php [hunk 4]
```diff
@@ -56,8 +53,6 @@ public function __construct(
 
     /**
      * @param \Doctrine\Common\Persistence\ObjectManager $manager
-     *
-     * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
```
Comment (TomasVotruba) [inline packages/framework/src/DataFixtures/DemoMultidomain/OrderDataFixture.php]: Btw, I'm curious, what way did you find most effective to remove these annotatoins?

Manually? Some cool regular expression or with [`GeneralPhpdocAnnotationRemoveFixer`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.11/src/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixer.php)? 

I'd probably go lazy manual way ;) but just in case you made some cool scripts, I'd like to see it. More PHP projects would be migrating this way, so this could be very useful.

File: UPGRADE.md [hunk 5]
```diff
@@ -117,7 +117,7 @@ in your product feed plugin because the instances of `StandardFeedItemInterface`
 
 ### [shopsys/coding-standards]
 #### From 3.x to 4.0
-- In order to run all checks, there is new unified way - execute `php vendor/bin/ecs check /path/to/project --config=vendor/shopsys/coding-standards/easy-coding-standard.neon`
+- In order to run all checks, there is new unified way - execute `php vendor/bin/ecs check /path/to/project --config=vendor/shopsys/coding-standards/easy-coding-standard.yml`
```
Comment (simara-svatopluk) [inline UPGRADE.md]: coding-standards version 4.0 was already released. You cannot change upgrade log of already released tag ;-)
So please revert these changes and create new section #### From [7.0.0-alpha2 to Unreleased]

File: CHANGELOG.md [hunk 6]
```diff
@@ -887,7 +887,8 @@ That's why is this section formatted differently.
     - rules configuration is now unified in single file - [`easy-coding-standard.neon`](./packages/coding-standards/easy-coding-standard.neon)
     - the option `ignore-whitespace` for rules checking method and class length is not available anymore
         - the limits were increased to 550 (class length) and 60 (method length)
-    
+- [#13](https://github.com/shopsys/coding-standards/) [EasyCodingStandard](https://github.com/Symplify/EasyCodingStandard) is now used
+    - rules config file changed its format from neon to yaml
```
Comment (TomasVotruba) [inline CHANGELOG.md]: It could be also useful to mention a version:

"[EasyCodingStandard](https://github.com/Symplify/EasyCodingStandard) v4.3 is now used "

File: build.xml [hunk 7]
```diff
@@ -6,70 +6,30 @@
...
     <property name="path.framework" value="${path.packages}/framework"/>
+    <property name="path.ecs.configuration" value="${path.packages}/coding-standards/easy-coding-standard.yml"/>
 
     <import file="project-base/build-dev.xml" />
     <import file="project-base/build.xml" />
 
-    <target name="standards" depends="phplint,phpcs-fixer,phpcs,phpmd,phpstan,twig-lint,yaml-lint,eslint-check,standards-packages" description="Checks coding standards in both project-base and packages."/>
-    <target name="standards-diff" depends="phplint-diff,phpcs-fixer-diff,phpcs-diff,phpmd-diff,phpstan,twig-lint-diff,yaml-lint,eslint-check-diff,standards-packages" description="Checks coding standards on changed files in project-base and all files in packages."/>
+    <target name="standards" depends="phplint,ecs,phpstan,twig-lint,yaml-lint,eslint-check,standards-packages" description="Checks coding standards in both project-base and packages."/>
+    <target name="standards-diff" depends="phplint-diff,ecs-fix,phpstan,twig-lint-diff,yaml-lint,eslint-check-diff,standards-packages" description="Checks coding standards on changed files in project-base and all files in packages."/>
```
Comment (simara-svatopluk) [inline build.xml]: not `ecs-fix`, but `ecs`. Because this target is about diffs; and if ecs cannot check diff, then let checkt it all.

Or can You write a target that checks `ecs-diff` simmilarly to old `phpcs-diff`?

File: build.xml [hunk 8]
```diff
@@ -6,70 +6,30 @@
...
-            <catch>
-                <phingcall target="phpcbf" />
-            </catch>
-        </trycatch>
+        <phingcall target="ecs-fix" />
         <phingcall target="eslint-fix" />
         <phingcall target="standards-fix-packages" />
     </target>
 
-    <target name="standards-fix-diff" description="Automatically fixes some coding standards violations in changed files in project-base and all files in packages.">
```
Comment (simara-svatopluk) [inline build.xml]: I don't see removing `standsrds-fix-diff` in PIA (yodiz), why did you remove this target? at least `eslint-fix-diff` and `standards-fix-packages` could stay here

File: composer.json [hunk 9]
```diff
@@ -94,6 +96,7 @@
     "doctrine/migrations": "^1.6.0",
     "doctrine/orm": "dev-doctrine-260-with-ddc1960-hotfix-and-ddc4005-hotfix as 2.6.0",
     "egeloen/ckeditor-bundle": "^4.0.6",
+    "friendsofphp/php-cs-fixer": "^2.10.0",
```
[the remark this answers is not in the document]
Comment (simara-svatopluk) [reply composer.json]: If you change it, do not forget to change it also in `conding-standards/composer.json`

File: packages/coding-standards/shopsys-coding-standard.yml [hunk 10]
```diff
@@ -0,0 +1,215 @@
...
+    ObjectCalisthenics\Sniffs\Files\ClassTraitAndInterfaceLengthSniff:
+        maxLength: 550
+    ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
+        maxLength: 60
+    ObjectCalisthenics\Sniffs\Metrics\PropertyPerClassLimitSniff:
+        maxCount: 30
+
+parameters:
+    exclude_files:
+        - '*tests/ShopBundle/Test/Codeception/_generated/AcceptanceTesterActions.php*'
```
Comment (simara-svatopluk) [inline packages/coding-standards/shopsys-coding-standard.yml]: This is `coding-standards` package, this file cannot exist here, so please remove maybe whole section `exclude_files`. But is has meaning in `project-base`, and you will have to deal with it somehow

Comment (simara-svatopluk) [reply packages/coding-standards/shopsys-coding-standard.yml]: This issue is many times in this file, so please move skips to appropriate packages

File: project-base/build-dev.xml [hunk 11]
```diff
@@ -409,10 +279,10 @@
 
     <target name="phpmd" description="Checks coding standards in all files by PHP Mess Detector.">
         <exec
-            executable="${path.phpmd.executable}"
-            logoutput="true"
-            passthru="true"
-            checkreturn="true"
+                executable="${path.phpmd.executable}"
```
Comment (simara-svatopluk) [inline project-base/build-dev.xml]: Please remove all remaining `phpcs`, `phpmd` and `phpcbf` from `build-dev.xml`

File: packages/coding-standards/README.md [hunk 12]
```diff
@@ -25,22 +25,22 @@ php composer require shopsys/coding-standards
...
 
-```neon
-#custom-coding-standard.neon
+```yaml
+#custom-coding-standard.yml
 includes:
-    - vendor/symplify/easy-coding-standard/config/psr2.neon
-    - vendor/shopsys/coding-standards/shopsys-coding-standard.neon
+    - vendor/symplify/easy-coding-standard/config/psr2.yml
+    - vendor/shopsys/coding-standards/shopsys-coding-standard.yml
```
[the remark this answers is not in the document]
Comment (boris-brtan) [reply packages/coding-standards/README.md]: thx

File: build.xml (hunk 7 above)
Comment (boris-brtan) [reply build.xml]: thx

File: packages/framework/src/DataFixtures/DemoMultidomain/OrderDataFixture.php (hunk 4 above)
Comment (boris-brtan) [reply packages/framework/src/DataFixtures/DemoMultidomain/OrderDataFixture.php]: mostly manually but when i saw some similiar parts i created just some simple rules. But still i needed to do it one by one and visual replace.

Comment (TomasVotruba) [reply packages/framework/src/DataFixtures/DemoMultidomain/OrderDataFixture.php]: I see, thanks for sharing

File: packages/coding-standards/README.md [hunk 13]
```diff
@@ -25,22 +25,22 @@ php composer require shopsys/coding-standards
...
 ## Usage
 
-Create `custom-coding-standard.neon` config file in your project which includes predefined rulesets. 
+Create `custom-coding-standard.yml` config file in your project which includes predefined rulesets. 
 You can also [customize the rules](./UPGRADE.md#version-4.0-and-higher) 
 and even add your own sniffs and fixers in the config.
 
-```neon
-#custom-coding-standard.neon
+```yaml
```
Comment (simara-svatopluk) [inline packages/coding-standards/README.md]: Shouldn't content looks like this?

```yaml
#custom-coding-standard.yml
imports:
    - { resource: 'vendor/symplify/easy-coding-standard/config/psr2.yml' }
    - { resource: 'vendor/shopsys/coding-standards/shopsys-coding-standard.yml' }
```

Well, I'm not sure how `yml` works, so thats why I ask :-)

File: packages/coding-standards/shopsys-coding-standard.yml [hunk 14]
```diff
@@ -0,0 +1,198 @@
...
+    ObjectCalisthenics\Sniffs\Files\ClassTraitAndInterfaceLengthSniff:
+        maxLength: 550
+    ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
+        maxLength: 60
+    ObjectCalisthenics\Sniffs\Metrics\PropertyPerClassLimitSniff:
+        maxCount: 30
+
+parameters:
+    exclude_files:
+        - '*tests/ShopBundle/Test/Codeception/_generated/AcceptanceTesterActions.php'
```
Comment (simara-svatopluk) [inline packages/coding-standards/shopsys-coding-standard.yml]: I'll repeat myself, because Tom reacted with :+1: , and maybe you thought it is solved ;-)

This is `coding-standards` package, this file cannot exist here, so please remove maybe whole section `exclude_files`. But is has meaning in `project-base`, and you will have to deal with it somehow

This issue is many times in this file, so please move skips to appropriate packages

File: packages/framework/.travis.yml [hunk 15]
```diff
@@ -16,8 +16,6 @@ install:
 
 script:
   - php vendor/bin/parallel-lint ./src
-  - php vendor/bin/php-cs-fixer fix --config=vendor/shopsys/coding-standards/build/phpcs-fixer.php_cs --dry-run --verbose --diff ./src
-  - php vendor/bin/phpcs --standard=vendor/shopsys/coding-standards/rulesetCS.xml --extensions=php --encoding=utf-8 --tab-width=4 -sp ./src
-  - php vendor/bin/phpmd "./src" text vendor/shopsys/coding-standards/rulesetMD.xml --extensions=php
+  - php vendor/bin/ecs check --config=vendor/shopsys/coding-standards/easy-coding-standard.yml --verbose ./src
```
Comment (simara-svatopluk) [inline packages/framework/.travis.yml]: Please add also `tests` to check

File: UPGRADE.md [hunk 16]
```diff
@@ -145,6 +145,26 @@ parameters:
 
 ```
 
+#### From [7.0.0-alpha2 to Unreleased]
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply UPGRADE.md]: I hope this will be automated soon. This is mess for everyone that should toolsourced.

File: packages/coding-standards/shopsys-coding-standard.yml (hunk 14 above)
Comment (boris-brtan) [reply packages/coding-standards/shopsys-coding-standard.yml]: thx.
moved to packages and project-base

File: packages/framework/autoload-coding-standard.yml [hunk 17]
```diff
@@ -0,0 +1,25 @@
+parameters:
```
Comment (simara-svatopluk) [inline packages/framework/autoload-coding-standard.yml]: I'm not sure about name of this file. In `coding-standards` package it is called `easy-coding-standard.yml`. I would prefer to name it the same.

File: packages/product-feed-heureka/.travis.yml [hunk 18]
```diff
@@ -16,7 +16,5 @@ install:
 
 script:
   - php vendor/bin/parallel-lint ./src ./tests
-  - php vendor/bin/php-cs-fixer fix --config=vendor/shopsys/coding-standards/build/phpcs-fixer.php_cs --dry-run --verbose --diff ./src ./tests
-  - php vendor/bin/phpcs --standard=vendor/shopsys/coding-standards/rulesetCS.xml --extensions=php --encoding=utf-8 --tab-width=4 -sp ./src ./tests
-  - php vendor/bin/phpmd "./src,./tests" text vendor/shopsys/coding-standards/rulesetMD.xml --extensions=php
+  - php vendor/bin/ecs check --config=vendor/shopsys/coding-standards/easy-coding-standard.yml --verbose ./src ./tests
```
Comment (simara-svatopluk) [inline packages/product-feed-heureka/.travis.yml]: I think this will fail because the config doesn't contain skips for this package.

I have a suggestion - change the command to
`php vendor/bin/ecs check --config=autoload-coding-standard.yml --verbose ./src ./tests`
And add import to `autoload-coding-standard`

Or maybe it is possbile to set multiple configs.

But it is maybe not possible because monorepo will include these `.yml` files... Not an easy task, maybe contact a college.
In this case it would be possible to have two configs in the package. One with exceptions (that is already here) and one with imports of `coding-standards` packages' config and exceptions config. But I would prefer to have only one config here, because two configs are strange.

Comment (simara-svatopluk) [reply packages/product-feed-heureka/.travis.yml]: And similar situation is in other packages

File: packages/coding-standards/shopsys-coding-standard.yml [hunk 19]
```diff
@@ -0,0 +1,143 @@
...
```
[the remark this answers is not in the document]
Comment (boris-brtan) [reply packages/coding-standards/shopsys-coding-standard.yml]: i have an idea to keep all the skips in shopsys-coding-standard.yml in package. This could solve maybe the problems with dependencies because of the name shopsys-coding-standard and others will still define theirs own rules due to instructions. Would be this solution ?

File: packages/product-feed-heureka/.travis.yml (hunk 18 above)
Comment (TomasVotruba) [reply packages/product-feed-heureka/.travis.yml]: > And add import to autoload-coding-standard

:+1:

File: UPGRADE.md [hunk 20]
```diff
@@ -17,6 +17,25 @@ There is a list of all the repositories maintained by monorepo, changes in log b
...
+    - { resource: '%vendor_dir%/shopsys/coding-standards/shopsys-coding-standard.yml' }
+parameters:
+    exclude_files:
+        - '*/ignored_folder/*'
+    skip:
+        # private properties should not start with "_"
+        PHP_CodeSniffer\Standards\Squiz\Sniffs\NamingConventions\ValidVariableNameSniff.PrivateNoUnderscore: ~
+        # skip max length of functions in files
+        ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
+            - '%current_working_dir%/src/file.php'
```
Comment (TomasVotruba) [inline UPGRADE.md]: This variable shortcut (`%current_working_dir%`) works only on `imports` section, see https://github.com/symplify/easyCodingStandard/#use-prepared-checker-sets

File: packages/coding-standards/README.md [hunk 21]
```diff
@@ -25,22 +25,23 @@ php composer require shopsys/coding-standards
...
 and even add your own sniffs and fixers in the config.
 
-```neon
-#custom-coding-standard.neon
-includes:
-    - vendor/symplify/easy-coding-standard/config/psr2.neon
-    - vendor/shopsys/coding-standards/shopsys-coding-standard.neon
+```yaml
+#custom-coding-standard.yml
+includes:imports:
```
Comment (TomasVotruba) [inline packages/coding-standards/README.md]: typo

File: project-base/build-dev.xml [hunk 22]
```diff
@@ -216,167 +216,37 @@
...
     </target>
 
-    <target name="phpcs-fixer-fix" description="Fixes coding standards violations in all files by PHP CS Fixer.">
+    <target name="ecs-fix" description="Checks and fixes automatically fixable coding standards in all files by PHP easy coding standards.">
         <exec
-            executable="${path.phpcs-fixer.executable}"
-            logoutput="true"
-            passthru="true"
-            checkreturn="false"
+                executable="${path.ecs.executable}"
```
Comment (TomasLudvik) [inline project-base/build-dev.xml]: There is too big indentation

**Comment by TomasVotruba** [comment]
> @TomasLudvik Config files should have same name, you use two types - easy-coding-standards.yml and autoload-coding-standard.yml please use one or another

Just a technical note (not sure what is the goal of these files):

- `easy-coding-standards.yml` is the format, that ECS loads by default (it's like `phpunit.xml` for PHPUnit)
- any other name have to be specified manually in CLI with `--config path/to/autoload-coding-standard.yml` option

File: packages/framework/autoload-coding-standard.yml (hunk 17 above)
Comment (boris-brtan) [reply packages/framework/autoload-coding-standard.yml]: to autoload-easy-coding-standards.yml

File: UPGRADE.md (hunk 20 above)
Comment (boris-brtan) [reply UPGRADE.md]: changed to */src

File: build.xml (hunk 7 above)
Comment (boris-brtan) [reply build.xml]: later found that ecs check --clear-cache could do the work

File: packages/coding-standards/README.md (hunk 13 above)
Comment (TomasVotruba) [reply packages/coding-standards/README.md]: @simara-svatopluk Here, it should.

`%vendor_dir%` is just variable to make it usable in case of different location, see [Symplify PR](https://github.com/Symplify/Symplify/pull/741))

File: packages/coding-standards/README.md [hunk 23]
```diff
@@ -31,9 +31,9 @@ and even add your own sniffs and fixers in the config.
 
 ```yaml
 #custom-coding-standard.yml
-includes:imports:
- - { resource: '%vendor_dir%/symplify/easy-coding-standard/config/psr2.yml' }
- - { resource: '%vendor_dir%/shopsys/coding-standards/shopsys-coding-standard.yml' }
+imports:
+   - { resource: '%vendor_dir%/symplify/easy-coding-standard/config/psr2.yml' }
+   - { resource: '%vendor_dir%/shopsys/coding-standards/shopsys-coding-standard.yml' }
```
Comment (TomasVotruba) [inline packages/coding-standards/README.md]: It's probably out of scope of this PR, but `psr2.yml` should be in `shopsys-coding-standard.yml`.

In the end, people load only one config (the Shopsys one).

See Sylius Coding Standard for inspiration: https://github.com/SyliusLabs/CodingStandard#installation--usage

File: CHANGELOG.md [hunk 24]
```diff
@@ -25,18 +25,115 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
     - `Front/Form/theme.html.twig`: block `checkbox_row` now uses block `form_label` for proper label rendering
         - the absence of `label` html tag was causing problems with JS validation (the error message was not included in the popup overview)
 
+### [shopsys/shopsys]
+#### Added
+- [#143](https://github.com/shopsys/shopsys/pull/143) [Shopsys Coding Standards dev-master](./packages/coding-standards/)  is now used
+    - config [easy-coding-standard.yml](./easy-coding-standard.yml]) for importing rules of new easy-coding-standard package 
+    - changed [composer.json](./composer.json) file 
+
+#### changed
```
Comment (TomasLudvik) [inline CHANGELOG.md]: pls update to upper C

File: CHANGELOG.md [hunk 25]
```diff
@@ -25,18 +25,115 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 #### Fixed
 - [#131 - correct rendering of checkbox label](https://github.com/shopsys/shopsys/pull/131):
     - `Front/Form/theme.html.twig`: block `checkbox_row` now uses block `form_label` for proper label rendering
         - the absence of `label` html tag was causing problems with JS validation (the error message was not included in the popup overview)
 
+### [shopsys/shopsys]
+#### Added
+- [#143](https://github.com/shopsys/shopsys/pull/143) [Shopsys Coding Standards dev-master](./packages/coding-standards/)  is now used
+    - config [easy-coding-standard.yml](./easy-coding-standard.yml]) for importing rules of new easy-coding-standard package 
+    - changed [composer.json](./composer.json) file
```
Comment (TomasLudvik) [inline CHANGELOG.md]: this is not important here, please remove it

File: CHANGELOG.md [hunk 26]
```diff
@@ -25,18 +25,115 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+### [shopsys/shopsys]
+#### Added
+- [#143](https://github.com/shopsys/shopsys/pull/143) [Shopsys Coding Standards dev-master](./packages/coding-standards/)  is now used
+    - config [easy-coding-standard.yml](./easy-coding-standard.yml]) for importing rules of new easy-coding-standard package 
+    - changed [composer.json](./composer.json) file 
+
+#### changed
+- [#143](https://github.com/shopsys/shopsys/pull/143) [Shopsys Coding Standards dev-master](./packages/coding-standards/)  is now used
+    - php-cs-fixer, phpcs, phpmd binaries replaced by ecs binaries
+    - build scripts to work with new easy-coding-standard checker
```
Comment (TomasLudvik) [inline CHANGELOG.md]: verb missing here

File: CHANGELOG.md [hunk 27]
```diff
@@ -25,18 +25,115 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+#### changed
+- [#143](https://github.com/shopsys/shopsys/pull/143) [Shopsys Coding Standards dev-master](./packages/coding-standards/)  is now used
+    - php-cs-fixer, phpcs, phpmd binaries replaced by ecs binaries
+    - build scripts to work with new easy-coding-standard checker 
+ 
+### [shopsys/coding-standards]
+##### Changed
+- [#143](https://github.com/shopsys/shopsys/pull/143) [EasyCodingStandard v4.3.0](https://github.com/Symplify/EasyCodingStandard/tree/4.3) is now used
+    - rules config file changed its format from neon to yaml
+    - modified composer.json file
```
Comment (TomasLudvik) [inline CHANGELOG.md]: please remove this

File: CHANGELOG.md [hunk 28]
```diff
@@ -25,18 +25,115 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
+### [shopsys/form-types-bundle]
+#### Changed
+- [#143](https://github.com/shopsys/shopsys/pull/143) [Shopsys Coding Standards dev-master](./packages/coding-standards/)  is now used
```
Comment (TomasLudvik) [inline CHANGELOG.md]: two spaces before is now used

File: CHANGELOG.md [hunk 29]
```diff
@@ -25,18 +25,115 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### [shopsys/framework]
+#### Changed
+- [#143](https://github.com/shopsys/shopsys/pull/143) [Shopsys Coding Standards dev-master](./packages/coding-standards)  is now used
```
[the remark this answers is not in the document]
Comment (TomasLudvik) [reply CHANGELOG.md]: two spaces before is now used - update this in all occurrences pls

File: CHANGELOG.md [hunk 30]
```diff
@@ -25,18 +25,115 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+    - version of coding-standards package to dev-master in [composer.json](./packages/framework/composer.json)
+    - modified [travis script]((./packages/framework/project-base/.travis.yml))
```
Comment (TomasLudvik) [inline CHANGELOG.md]: why do you use two braces here? update in all occurrences pls

**Comment by TomasVotruba** [comment]
@boris-brtan This was a huge challange for first pull-request. I appreciate it, great job!
